app/helpers/view_components/user_menu.rb

```diff
@@ -53,7 +53,10 @@ def profile_item
         avatar(current_user, alt: "Your uploaded avatar") +
           tag.div(class: 'info') do
             tag.div(current_user.name, class: 'name') +
-              tag.div("@#{current_user.handle}", class: 'handle')
+              tag.div(class: "handle flex") do
+                tag.span("@") + render(ViewComponents::HandleWithFlair.new(current_user.handle, current_user.flair,
+                  size: 14)).html_safe
+              end
```

Comment:
What happens when the user does not have any flair? I looks like the `@` is still rendered. Is that correct?

Reply:
yes, that will just show the handle with a @ before it. this is the user menu, where we currently show the handle with a @ before it

Reply:
Okay great!